internal/cel/library/oci.go

```diff
@@ -50,6 +50,23 @@ import (
 // Examples:
 //
 //	kw.oci.image('image:latest').manifestDigest() // returns the digest of the image manifest
+// 
+// manifest_config
+//
+// Returns the manifest, digest and image configuration of the OCI image.
+// See more information about the fields available of manifest and image
+// configuration at:
+// https://github.com/opencontainers/image-spec/blob/main/manifest.md
+// https://github.com/opencontainers/image-spec/blob/main/config.md
+//
+//	<OCIClient>.manifest_config() <DynamicMap>
+//
+// Examples:
+//
+//	kw.oci.image('image:latest').manifest_config().manifest // returns the image manifest
+//	kw.oci.image('image:latest').manifest_config().config // returns the image configuration
+//	kw.oci.image('image:latest').manifest_config().digest // returns the image digest
+
```

Comment:
this extra line needs to be removed or the godoc will not recognize this comment as a docstring